src/ResourceManager/Resources/Commands.Resources/Models.ResourceGroups/ResourceClient.ResourceManager.cs

```diff
@@ -47,55 +42,6 @@ public partial class ResourcesClient
         /// </summary>
         /// <param name="parameters">The get parameters</param>
         /// <returns>List of resources</returns>
-#if !NETSTANDARD
-        public virtual List<PSResource> FilterPSResources(BasePSResourceParameters parameters)
```

Comment:
This isn't used anywhere?

Reply:
Same comment as above: this was based on the old ResourceManager SDK.